api.md

```diff
@@ -431,12 +434,104 @@ PostGIS geospatial indexes).  During testing, you might check your
+| `data_nodes` | This is the set of data nodes that will be used for this table.  If this is not present then all data nodes known by this instance will be used to distribute the hypertable. | 
+| `chunk_target_size` | DEPRECATED - The target size of a chunk (including indexes) in `kB`, `MB`, `GB`, or `TB`. Setting this to `estimate` or a non-zero chunk size, e.g., `2GB` will enable adaptive chunking (a DEPRECATED feature). The `estimate` setting will estimate a target chunk size based on system information. Adaptive chunking is disabled by default. |
+| `chunk_sizing_func` | DEPRECATED - Allows setting a custom chunk sizing function for adaptive chunking (a DEPRECATED feature). The built-in chunk sizing function will be used by default. Note that `chunk_target_size` needs to be set to use this function.  |
+
+#### Returns
+
+|Column|Description|
+|---|---|
+| `hypertable_id` | ID of the hypertable in TimescaleDB. |
+| `schema_name` | Schema name of the table converted to hypertable. |
+| `table_name` | Table name of the table converted to hypertable. |
+| `created` | True if the hypertable was created, false when `if_not_exists` is true and no hypertable was created. |
+
+#### Sample Usage [](create_distributed_hypertable-examples)
+
+Create a table `conditions` which will be partitioned across data
+nodes by the 'location' column.  Note that the number of space
+partitions is automatically equal to the number of data nodes in this
+case.
+```sql
+SELECT create_distributed_hypertable('conditions', 'time', 'location');
+```
+
+Create a table `conditions` using a specific set of data nodes.
+```sql
+SELECT create_distributed_hypertable('conditions', 'time', 'location',
+    data_nodes => '{ "data_node_1", "data_node_2", "data_node_4", "data_node_7" }');
+```
+
+#### Best Practices [](create_distributed_hypertable-best-practices)
+
+**Space partitions:** As opposed to the normal [`create_hypertable` best practices](#create_hypertable-best-practices),
+space partitions are highly recommended for distributed hypertables.
+Incoming data will be divided among data nodes based upon the space
+partition (the first one if multiple space partitions have been
+defined).  If there is no space partition, all the data for each time
+slice will be written to a single data node.
+
+**Time intervals:** Follow the same guideline in setting the `chunk_time_interval`
+as with [`create_hypertable`](#create_hypertable-best-practices),
+bearing in mind that the calculation needs to be based on the memory
+capacity of the data nodes.  However, one additional thing to
+consider, assuming space partitioning is being used, is that the
+hypertable will be evenly distributed across the data nodes, allowing
+a larger time interval.  In this case you can consider the total
+memory across all the data nodes, though the incoming rate should also
+be multiplied by the `replication_factor` if greater than 1.
```

Comment:
Again, I think we should not mention replication_factor here. Let's add that when the feature is ready.

Reply:
This still feels a bit like we're setting the users up to fail if they play around with the replication_factor rather than simply preventing them from doing so.  Removing this line, but we need to make this more coherent before public beta.